### PR TITLE
feat: display link to the launch in the launch admin page

### PR DESCRIPTION
**Pull request by wantedsystem**

### CODE REVIEW

File: src/admin/lbps/dashboard/dashboard.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
           </div>
         </div>
       </div>
+      <a href.to-view="lbp/${selectedLbp.address}">Go to LBP Dashboard</a>
```
Comment (dkent600) [inline src/admin/lbps/dashboard/dashboard.html:14]: Do these links show if there is no selected token?  They should not.

**Review by dkent600 — CHANGES_REQUESTED** [review]
Please provide a screenshot

**Review by dkent600 — CHANGES_REQUESTED** [review]
Can you please supply a screenshot?  THanks

**Comment by wantedsystem** [comment]
Results =>

Capture d’écran 2021-11-08 à 18 38 21

**Review by dkent600 — APPROVED** [review]

**Review by dkent600 — APPROVED** [review]

**Pull request merged**